lib/serial_terminal.pm

```diff
@@ -22,7 +22,6 @@ BEGIN {
       add_serial_console
       get_login_message
       login
-      select_virtio_console
```

Comment:
what the reason to move a function with name select_serial_terminal from file with name serial_terminal ?

Reply:
@asmorodskyi Just a quick reply as I agree to get back to this after the release. My goal was to have general function which provides user terminal/terminal emulator/ssh connection/whatever, so one can run shell based commands (no X / GUI) *on any backend*. If you look into code repository, there is many if clauses with select_console => create wrapper to this function and move if clauses there. The benefit is, that code is ready to be run on any backend. Does it makes sense?
I changed names from `select_text_console()` to `select_serial_terminal()` on @richiejp request, but not sure about it, as it can be confusing. I want create more general approach than just virtio console wrapper.